packages/database/database.go

```diff
@@ -1,134 +1,65 @@
 package database
 
 import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
 	"sync"
-	"time"
 
-	"github.com/dgraph-io/badger"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/iotaledger/goshimmer/packages/parameter"
+	"github.com/iotaledger/hive.go/database"
 )
 
 var (
-	ErrKeyNotFound = badger.ErrKeyNotFound
-
-	dbMap = make(map[string]*prefixDb)
-	mu    sync.Mutex
+	instance *badger.DB
+	once     sync.Once
 )
 
-type prefixDb struct {
-	db     *badger.DB
-	name   string
-	prefix []byte
-}
-
-func getPrefix(name string) []byte {
-	return []byte(name + "_")
-}
-
-func Get(name string) (Database, error) {
-	mu.Lock()
-	defer mu.Unlock()
-
-	if db, exists := dbMap[name]; exists {
-		return db, nil
-	}
-
-	badger := GetBadgerInstance()
-	db := &prefixDb{
-		db:     badger,
-		name:   name,
-		prefix: getPrefix(name),
-	}
-
-	dbMap[name] = db
-
-	return db, nil
-}
+func GetGoShimmerBadgerInstance() *badger.DB {
```

Comment:
Consider renaming to `GetBadgerInstance()`

Reply:
Can't, as aliasing the database package type doesn't work completely. Having the function named like this gives a better indication when the wrong database package is used.

Reply:
Sure, this should only be renamed when database package is aliased

Reply:
Changed